app/controllers/reports_controller.rb

```diff
@@ -103,11 +103,11 @@ def compute_rpm(count, total_reads)
     end
   end
 
-  def compute_zscore(count, mean, stdev)
-    if count && mean
-      (count - mean) / stdev
-    elsif count
-      Float::MAX
+  def compute_zscore(rpm, mean, stdev)
+    if rpm && mean
```

Comment:
isn't checking  stdev more intuitive than mean? since you are checking for divide by zero

Reply:
agree